package.json

```diff
@@ -56,6 +56,7 @@
   },
   "dependencies": {
     "@tinymce/tinymce-react": "^3.13.0",
+    "tinymce": "^5.10.2",
```

Comment:
We need the non-react wrapper version of tinymce to use plugins. Thankfully both of these dependancies will be removed in facvor of paragon soon.